ArduCopter/sensors.cpp

```diff
@@ -166,6 +166,7 @@ void Copter::read_battery(void)
     // update motors with voltage and current
     if (battery.get_type() != AP_BattMonitor::BattMonitor_TYPE_NONE) {
         motors->set_voltage(battery.voltage());
+        AP_Notify::flags.battery_voltage = battery.voltage(); //for on-board oled display
```

Comment:
I like the comments - makes it much clearer.  I'm not sure how its obvious that 
AP_Notify::flags.battery_voltage = battery.voltage();
is for the oled only but perhaps I'm missing something.